inductiva/molecules/utils.py

```diff
@@ -47,7 +46,7 @@ def align_trajectory_to_average(universe, trajectory_output_path):
                     in_memory=False).run()
 
 
-def download_pdb_from_rcsb(pdb_id: str, save_dir: Optional[str] = None) -> str:
+def download_pdb_from_rcsb(pdb_id: str) -> str:
```

Comment:
Why did you remove save_dir?

If so you need to remove it in the docstring as well.

Reply:
It will be by default, to not add extra complexity to the download.